render/ascii.go

```diff
@@ -14,6 +14,7 @@ const (
 	ASCIISnakeHead = "H"
 	ASCIISnakeBody = "O"
 	ASCIISnakeTail = "T"
+	ASCIIHazard    = "."
```

Comment:
I added support for hazards in the ascii renderer. Seems to look decent.

![image](https://user-images.githubusercontent.com/969644/151723042-a2ea9c5f-db77-413c-b0d2-02e48e79e2bf.png)
![image](https://user-images.githubusercontent.com/969644/151723060-1059af2f-a2bc-41b0-bb8e-5946c49d3277.png)

Reply:
ASCII can't support overlaps, so I went with the strategy of only displaying the hazard if it's the only thing in the square.

I tried a few symbols to represent hazards, like `#`, `x` and `@` and I found that `.` was best because it didn't make things too noisy.

Reply:
Yeah, I think this looks great!